bread/layout/components/forms/widgets.py

```diff
@@ -28,9 +29,9 @@ class BaseWidget(hg.DIV):
     django_widget = None
 
     # default attributes which are used to create the input element in a standardized way for many inputs
-    carbon_input_class = ""
-    carbon_input_error_class = ""
-    input_type = None
+    carbon_input_class: Optional[str] = ""
+    carbon_input_error_class: Optional[str] = ""
+    input_type: Optional[str] = None
```

Comment:
According to https://docs.python.org/3/library/typing.html#typing.Optional this should be ```carbon_input_class: str = ""``` because 
> Optional[X] is equivalent to X | None (or Union[X, None]).

Reply:
(Only for ```carbon_input_class``` and ```carbon_input_error_class```)